test/visual-diff/visual-tests

```diff
@@ -625,5 +625,14 @@
       "loading_complete_delay_ms": 1000,
       "interactive_tests": "examples/visual-tests/amp-accordion/amp-accordion.js"
     },
+    {
+      "url": "examples/visual-tests/amp-user-notification/amp-user-notification.amp.html",
+      "name": "amp-user-notification",
+      "viewport": {"width": 400, "height": 600},
+      "loading_complete_css": [
```

Comment:
@danielrozenberg Do you think it would be better practice to have test functionality all in the JS, or a mix between config and their JS. 

I think we could move this into the JS file, and use the `verifyCss...` function right? I feel like that would be slightly cleaner 😄

Reply:
They serve a different purpose - the JS code doesn't start running until after the `loading_complete_css` is verified; that means you should use `loading_complete_css` to mark when the page is ready for interactions, and move forward from there

Reply:
Ahh. Sounds good! Thank you for the explanation! 😄